packages/documentation/pages/usage/components/form-fields.vue

```diff
@@ -1127,6 +1193,7 @@ export default defineComponent({
 				numberMaximum: null,
 				numberMinimum: null,
 				numberStep: null,
+				shortcuts: null,
```

Comment:
shouldn't it be `[]`

Reply:
For the purpose of the docs, this `shortcuts` is a `KtFieldToggleGroup` and not the actual KtFieldDate's `shortcuts` prop. Its value is `Record<string, boolean | null> | null`